File: leetcode-questions/csharp/src/test/csharp/com/leetcode/questions/reverseinteger/ReverseIntegerTest.cs
using Xunit;
using System.Collections.Generic;
using com.leetcode.questions.leetcode.editor.en.reverseinteger;
using com.leetcode.questions.support;

namespace com.leetcode.questions.reverseinteger;

public class ReverseIntegerTest {

    private const string CasesFileName = "reverse-integer.json";

    public ReverseIntegerTest() {

    }

    [Fact]
    public void TestCases() {
        GeneratedLeetCodeTestHarness.Run(typeof(Solution), "Reverse", GeneratedLeetCodeTestHarness.ReadCasesFile(CasesFileName));
    }
}

File: leetcode-questions/java/src/test/java/com/leetcode/questions/reverseinteger/ReverseIntegerTest.java
package com.leetcode.questions.reverseinteger;

import com.leetcode.questions.leetcode.editor.en.ReverseInteger;
import com.leetcode.questions.support.GeneratedLeetCodeTestHarness;
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;
import java.nio.file.Files;
import java.nio.file.Path;

public class ReverseIntegerTest {

    private static final Path CASES_PATH = Path.of("generated-test-cases", "reverse-integer.json");

    public ReverseIntegerTest() {

    }

    @Test
    public void testCases() throws Exception {
        GeneratedLeetCodeTestHarness.run(ReverseInteger.Solution.class, "reverse", Files.readString(CASES_PATH));
    }
}
